File: src/main/python/Person.py
class Person:
    def __init__(self, name=None, hobbies=None):
        self.name = name
        self.hobbies = hobbies if hobbies is not None else []

    def __str__(self):
        return f"Person(name={self.name}, hobbies={self.hobbies})"

File: src/main/java/Person.java
import com.thoughtworks.xstream.annotations.XStreamAlias;
import lombok.*;

import java.util.List;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@ToString
@XStreamAlias("person")
public class Person {
    @XStreamAlias("Name")
    private String name;
    @XStreamAlias("Hobbies")
    private List<String> hobbies;
}
